queries/djot/highlights.scm

```diff
@@ -0,0 +1,276 @@
+(heading1) @markup.heading.1
+
+(heading2) @markup.heading.2
+
+(heading3) @markup.heading.3
+
+(heading4) @markup.heading.4
+
+(heading5) @markup.heading.5
+
+(heading6) @markup.heading.6
+
+(thematic_break) @string.special
+
+[
+  (div_marker_begin)
+  (div_marker_end)
+] @punctuation.delimiter
+
+[
+  (code_block)
+  (raw_block)
+  (frontmatter)
+] @markup.raw
```

Comment:
```suggestion
([
  (code_block)
  (raw_block)
  (frontmatter)
] @markup.raw
  (#set! "priority" 90))
```

Reply:
What's the reason for priority here? What changes?

Reply:
The priority, to make sure the `@markup.raw` is overriden by anything(!) that is injected.

Reply:
Ah, thank you! That makes sense.